composer.lock

```diff
@@ -741,27 +741,29 @@
         },
         {
             "name": "doctrine/instantiator",
-            "version": "1.1.0",
```

Comment:
The entire diff is the result of `cu --with-dependencies phpunit/phpunit` - I didn't want to update any other dependencies.